etsin_finder/frontend/js/components/qvain/files/metadataModal/options.js

```diff
@@ -0,0 +1,48 @@
+import translate from 'counterpart'
+
+export const getOptions = () => ({
```

Comment:
Good that you separated this into its own functions, makes the code more clean and intuitive.